inputfiles/idl/HTML - Links.commentmap.json

```diff
@@ -0,0 +1,13 @@
+{
+    "hyperlink-href": "Returns the hyperlink's URL.\nCan be set, to change the URL.",
```

Comment:
i do not see these comments showing up

Reply:
Not sure what should be done there, it seems should be `htmlhyperlinkelementutils-href` instead of `hyperlink-href` to add it to the lib file. Should it be manually mapped?